packages/docsearch-react/src/DocSearchModal.tsx

```diff
@@ -54,6 +54,7 @@ export function DocSearchModal({
   const searchBoxRef = React.useRef<HTMLDivElement | null>(null);
   const dropdownRef = React.useRef<HTMLDivElement | null>(null);
   const inputRef = React.useRef<HTMLInputElement | null>(null);
+  const modalRef = React.useRef<HTMLInputElement | null>(null);
```

Comment:
```suggestion
  const modalRef = React.useRef<HTMLDivElement | null>(null);
```

You can move this line after `containerRef` to respect the UI structure.